src/v2/Apps/Feature/Components/FeatureHeader/FeatureHeaderDefault.tsx

```diff
@@ -45,46 +42,47 @@ export interface FeatureHeaderDefaultProps {
 export const FeatureHeaderDefault: React.FC<FeatureHeaderDefaultProps> = ({
   feature: { name, subheadline, defaultImage: image },
 }) => {
-  return (
-    <Container
-      display={["block", "flex"]}
-      height={[
-        "auto",
-        !!image ? `calc(95vh - ${DESKTOP_NAV_BAR_HEIGHT}px)` : "50vh",
-      ]}
-    >
-      {image && (
-        <Figure height={["50vh", "auto"]}>
-          <picture>
-            {/* @ts-expect-error STRICT_NULL_CHECK */}
-            <source srcSet={image.sm.srcSet} media="(max-width: 400px)" />
-            {/* @ts-expect-error STRICT_NULL_CHECK */}
-            <source srcSet={image.md.srcSet} media="(max-width: 1200px)" />
-            {/* @ts-expect-error STRICT_NULL_CHECK */}
-            <source srcSet={image.lg.srcSet} media="(min-width: 1200px)" />
-            {/* @ts-expect-error STRICT_NULL_CHECK */}
-            <Image src={image.sm.src} alt={name} loading="lazy" />
-          </picture>
-        </Figure>
-      )}
+  if (image) {
+    return (
+      <FullBleed>
+        <Container
+          display={["block", "flex"]}
+          height={[
+            "auto",
+            !!image ? `calc(95vh - ${DESKTOP_NAV_BAR_HEIGHT}px)` : "50vh",
+          ]}
+        >
+          <Figure height={["50vh", "auto"]} backgroundColor="black10">
+            <picture>
+              <source srcSet={image.sm?.srcSet} media="(max-width: 400px)" />
+              <source srcSet={image.md?.srcSet} media="(max-width: 1200px)" />
+              <source srcSet={image.lg?.srcSet} media="(min-width: 1200px)" />
+              <Image src={image.sm?.src} alt={name} loading="lazy" />
+            </picture>
+          </Figure>
 
-      <Meta p={4} flexBasis={image ? "50%" : "100%"}>
-        <Join separator={<Spacer my={1} />}>
-          <Text
-            variant="largeTitle"
-            as="h1"
-            fontSize="size10"
-            textAlign="center"
-          >
-            {name}
-          </Text>
+          <Meta p={4} flexBasis={image ? "50%" : "100%"}>
+            <Text
+              variant="largeTitle"
```

Comment:
This is no longer a valid variant under v3. Should be `xl` or `xxl`

Reply:
Here's a list of all of the Typography options and how they map between v2 and v3: https://palette.artsy.net/atoms/typography/

Reply:
Here's the mapping between, with a migration guide: https://palette.artsy.net/guides/migration/